rest/taskana-rest-spring-example-boot/src/test/pro/taskana/rest/ExampleRestApplicationTest.java

```diff
@@ -0,0 +1,14 @@
+package pro.taskana.rest;
```

Comment:
why does this test class exist? Honestly I dont see any tests coming soon for this module

Reply:
removed test classes and test dependencies